aplib.net-demo/Assets/Scripts/Items/Items/TestItem2.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TestItem2 : Item
+{
+
+    //You want to change the value of the uses and all the attributes of the item in the unity inspector for the item prefab you make
```

Comment:
This is a comment on nothing for some functionality that does not exist, either remove it or implement the functionality it describes.